src/fsharp/service/ServiceUntypedParse.fsi

```diff
@@ -22,11 +22,20 @@ type public FSharpParseFileResults =
     /// Notable parse info for ParameterInfo at a given location
     member FindNoteworthyParamInfoLocations : pos:pos -> FSharpNoteworthyParamInfoLocations option
 
+    /// Gets the ranges of all arguments, if they can be found, for a function application at the given position.
+    member GetAllArgumentsForFunctionApplicationAtPostion: pos: pos -> range list option
+
+    /// Determines if the expression or pattern at the given position has a type annotation
+    member IsTypeAnnotationGiven: pos -> bool
+
+    /// Determines if the binding at a given position is a lambda expression
```

Comment:
What's meant by `position` here?  The position of the identifier for the value being bound?

Reply:
Yep, this check if `x` in `let x = ...` binds to a lambda. Will adjust the name.